app/src/main/res/layout/activity_workout_analytics.xml

```diff
@@ -0,0 +1,294 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".frontend.WorkoutAnalytics">
+
+    <Spinner
+        android:id="@+id/measurements"
```

Comment:
On the floating team android phone, the words are slightly cut off by the screen.